SimPEG/Problem.py

```diff
@@ -263,7 +262,7 @@ class LinearProblem(BaseProblem):
 
     # surveyPair = Survey.LinearSurvey
 
-    G = None
+    F = None
```

Comment:
what is the motivation for changing this from `G` to `F`? (in a lot of geophysical literature, the canonical linear problem is written as `Gm=d`)